geometries/brian/M82Geometry.py

```diff
@@ -1,30 +1,27 @@
+import os
 import spiceypy as sp
 
 def main():
 
-    mk = '../kernels/asperaMetaKernel.tm'
-    spk = 'NewM82.bsp'
-    pck = 'NewM82.tpc'
+    mk = os.path.join(os.path.dirname(__file__),'../kernels/mk/asperaMetaKernel.tm')
+    spk = os.path.join(os.path.dirname(__file__),'M82spk.bsp')
 
     sp.furnsh(mk)
     sp.furnsh(spk)
-    sp.furnsh(pck)
 
     targ = 9999000
```

Comment:
this could be **targ = 'M82'**, and below could be **obs = 'EARTH'**, then SPK call could use **sp.spkpos**